doc/sample_worksets.yaml

```diff
@@ -0,0 +1,47 @@
+name: Native-authored Workset
+id: 6408a5e930000067055ce2e8
+volumes: 919
+description: >
+             A list of works authored
+             by Native Americans in HathiTrust
+author: SCWAReD
```

Comment:
```suggestion
- name: Native-authored Workset
  id: 6408a5e930000067055ce2e8
  volumes: 919
  description:
             A list of works authored
             by Native Americans in HathiTrust
  author: SCWAReD
```

Reply:
Please write the code in such a way that this file could be fetched from anywhere ... locally, or remotely.
(i.e. use a url to reference the file, like `file:///blah.yaml` which could be `http://foo/blah.yaml`)

Reply:
Good idea